common/rnc_deco.h

```diff
@@ -19,15 +19,13 @@
  *
  */
 
-#ifndef SKY_RNC_DECO_H
-#define SKY_RNC_DECO_H
+#ifndef COMMON_RNC_DECO_H
+#define COMMON_RNC_DECO_H
 
 
+namespace Common {
 
-#define RNC_SIGNATURE   0x524E4301 // "RNC\001"
-
-namespace Sky {
-
+// Decoder for RNC (Rob Norton Computing) ProPack compression.
```

Comment:
```suggestion
// Decoder for RNC (Rob Northen Computing) ProPack compression.
```
https://wiki.multimedia.cx/index.php/RNC_ProPack